configuration.md

```diff
@@ -21,5 +22,39 @@ The DSpace server will have a whitelist of properties that can be retrieved by t
 }
 ```          
 
-* 200 OK - if the operation succeed
-* 404 Not found - if the property doesn't exist or isn't configured to be retrieved
\ No newline at end of file
+* 200 OK - if the operation succeeded
+* 404 Not found - if the property doesn't exist or isn't configured to be exposed
+
+## Search methods
+**/api/config/properties/search/isExposed**
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+
+This endpoint returns all exposed configuration variables.
+
+For example:
+
+```json
+[{
```

Comment:
we need to follow the usual HAL format for resource collections having an embedded with the pagination information